jib-cli/src/main/java/com/google/cloud/tools/jib/cli/CommonContainerConfigCliOptions.java

```diff
@@ -176,4 +178,20 @@
     }
     return Optional.empty();
   }
+
+  /**
+   * Returns {@code true} if the user-specified base image is jetty or if a custom base image is not
+   * specified.
+   *
+   * @return a boolean
+   * @throws InvalidImageReferenceException if image reference is invalid
+   */
+  public Boolean isJetty() throws InvalidImageReferenceException {
```

Comment:
Sorry, missed this. I think the name should be `isJettyBaseImage`. Otherwise, it's confusing.